.rspec

```diff
@@ -1,3 +1,5 @@
+--require manageiq/spec/spec_helper
 --require spec_helper
 --color
 --order random
+--exclude-pattern "spec/manageiq/**/*_spec.rb"
```

Comment:
Are the main repo tests always run right now? O_o

Reply:
I'm not really sure.  I had to add this for the other plugins so that they wouldn't run the manageiq specs, so I'm assuming they are running, but didn't check.